packages/frontend/src/components/global/MkStickyContainer.vue

```diff
@@ -49,50 +49,60 @@ const calc = () => {
 		childStickyBottom.value = parentStickyBottom.value + footerEl.value.offsetHeight;
 		footerHeight.value = footerEl.value.offsetHeight.toString();
 	}
-};
+});
 
-const observer = new ResizeObserver(() => {
-	window.setTimeout(() => {
-		calc();
-	}, 100);
+const ro = new ResizeObserver(() => {
+	calc();
 });
 
+const cleanups: (() => void)[] = [];
+const cleanup = () => {
+	for (const cl of cleanups) cl();
```

Comment:
要素数が少ないので大きな差はないし特にコーディングルールがある認識でもないので好みの問題かもしれないですが
```suggestion
	cleanups.forEach(cl => cl());
```

Reply:
個人的にはforEachで回す方が好みですが、既にfor...ofで実装されているコードがあったので、そちらに合わせました
https://github.com/misskey-dev/misskey/blob/08d618bb8b98199a4670313019d6a85ad4cf155b/packages/frontend/src/scripts/get-note-menu.ts#L454
https://github.com/misskey-dev/misskey/blob/08d618bb8b98199a4670313019d6a85ad4cf155b/packages/frontend/src/scripts/get-user-menu.ts#L387